### ISSUE TITLE
Impossibility to go right (sometimes)

### ISSUE THREAD

**Issue by AnddyAnddy** [haxball/haxball-issues#1211]
No matter the browser (chrome, firefox, opera), no matter the assigned keys (right, d, other random stuff), i experience sometimes the impossibility of going to the right direction in other rooms than mine.
This happens ONLY with the right direction, and it seems like it started to happen 2-3 weeks ago, I thought that i was the only one, but I saw other people experiencing my issue :(

**Comment by basro** [comment]
I can't think of any reason why the right movement in particular would stop working.
Also, there have been no changes to haxball in the last 2-3 weeks and since you say it happens in all browsers I think it's very unlikely that this is a new bug.

When this happens, for how long are you unable to move right?

In firefox if the app loses focus while you are holding down a key that key will get stuck until you press and release it again, could that be what happened to you? If the left key gets stuck the right key will seem to not work until you unstuck the left key.

You can do this by accident very easily for example pressing the alt key while holding left.

**Comment by AnddyAnddy** [comment]
When it happens, I just have to release and press right again, it's like I never pressed it, If I keep pressing right it will never go back to normal 


It can happen when I do press left, then right, then release left (I should go right but I don't), but it does happen also when no other keys are pressed, moreover, I tried playing with arrows and 4 others setup and I still have this issue

And as far as I tried, I couldn't reproduce the bug on my own host whereas it happens ~randomly around 4 and 10 times per min on vps

**Comment by basro** [comment]
if you can capture this in a replay recording and send it to me it'd be very useful.

**Comment by coduh** [comment]
Indeed, this happens to me too. I thought it was my old keyboard.. lol
The right arrow is just stuck so i have to release and press again.

**Edit**: I think it happens after typing something in the chat or pressing Esc, even though i release & press, it detects a false release.

**Comment by AnddyAnddy** [comment]
I haven't had this issue for a while, somehow the issue vanished for me, no idea about others

**Comment by AnddyAnddy** [comment]
It happened again !

https://www.haxball.com/replay?v=3#https://www.haxballclub.com/hbr/haxball-replay-95094.hbr2
0:57
after I pass to VAR, i'm trying to go to the right, but I just can't, i'm forced to press it again

https://www.haxball.com/replay?v=3#https://www.haxballclub.com/hbr/haxball-replay-14983.hbr2
at the kick off
Pedy passes to me, I logically press right and can't move, i'm trying to press down too and I moved down, I had to press several times right to be able to go right 
it happened several times during this particular replay but this is the most obvious case
